the-badass-todo-list-part-1/main.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
   // Kick off
+  // Quiz#3
```

Comment:
Good point. We have some similar methods in Gaia. For example:

https://github.com/mozilla-b2g/gaia/blob/master/apps/system/lockscreen/js/lockscreen_state_manager.js#L481

Although to deal with real bootstrapping flow, you need some more delicate and complete solutions.